<launch>
  <!-- Launch turtlebot_pose node -->
  <include file="$(find turtlebot_pose)/launch/turtlebot_pose.launch" />

  <!-- Launch turtlebot_scan node -->
  <include file="$(find turtlebot_scan)/launch/turtlebot_scan.launch" />

  <!-- Launch turtlebot_controller node -->
  <node pkg="turtlebot_controller" type="turtlebot_controller_node" name="turtlebot_controller_node" output="screen">
    <param name="theta_Kp" value="0.5" />
    <param name="theta_Ki" value="0.00001" />
    <param name="theta_Kd" value="1.8" />
    <param name="velocity_Kp" value="1.1" />
    <param name="velocity_Ki" value="0.00001" />
    <param name="velocity_Kd" value="0.5" />
  </node>
</launch>
